_source/molecules/empty/Empty.scss

```diff
@@ -6,12 +6,21 @@
   align-items: center;
   max-width: 350px;
   text-align: center;
+  color: $color__font--dark;
 
   @include mobile {
     padding: 0 calc(3% + 6px);
   }
 }
 
+.empty--dark-mode {
```

Comment:
Can we keep it consistent? Sometimes the modifier for the dark-mode is dark, sometimes it's dark-mode (which I prefer, especially if "dark" is white).